src/api/VoteMonitor.Api.Observer/Commands/RemoveDeviceIdCommand.cs

```diff
@@ -0,0 +1,21 @@
+﻿using MediatR;
+using AutoMapper;
+using VoteMonitor.Api.Observer.Models;
+
+namespace VoteMonitor.Api.Observer.Commands
+{
+    public class RemoveDeviceIdCommand : IRequest<bool>
+    {
+        public int IdObserver { get; set; }
+    }
+
+    public class RemoveDeviceIdProfile : Profile
+    {
+        public RemoveDeviceIdProfile()
+        {
+            CreateMap<RemoveDeviceIdModel, RemoveDeviceIdCommand>()
+                .ForMember(dest => dest.IdObserver, c => c.MapFrom(src => src.IdObserver))
```

Comment:
This mapping definition (IdObserver) is not needed as AutoMapper already solves it by convention (the property names are the same)